package.json

```diff
@@ -25,7 +25,7 @@
     "activationEvents": [
         "*"
     ],
-    "main": "./src/kite",
+    "main": "./dist/kite-extension",
```

Comment:
It looks like the CI has trouble detecting this directory (https://travis-ci.org/kiteco/vscode-plugin/builds/521011572#L522). Is that something that needs to be updated elsewhere?

Reply:
Yeah, I noticed that too, and am not really sure.
Weird thing is is that the extension runs fine when I build it locally for production and install it - which to me indicates that there's something the `launch.json` that's off, or otherwise with the testing setup.
Further weirdness, is that we're still having the exact same unit test passing / failing profile, even though it's complaining about the module loading...

Reply:
Would this cause any potential problems with our release pipelines? If not, let's just create an issue to track this as a follow up.

Reply:
Nope. Test with this line / script on your machine if you want to verify: https://github.com/kiteco/vscode-plugin/pull/224/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R96

Reply:
Quick aside on the script itself; would it be worth the effort to add a check to see if `vsce` is present in the user's machine / PATH, and then to install it if not?

Reply:
`vsce` isn't needed on user machines. Its utility for us is purely devops and development. We use it to publish the plugin on Solness and to package to test installation locally via the script I made for this PR